### ISSUE TITLE
Download Cloudbuild firmwares

### ISSUE THREAD

**Issue by raphaelcoeffic** [EdgeTX/buddy#67]
When using the regular "Cloud" tab, the firmware for a specific radio can be downloaded. This option does not exist yet for the firmwares built via "Cloudbuild", but would be very useful IMHO.

**Comment by freshollie** [comment]
Yes, this was understood to be a missing feature. The flashing method was faster to implement because flashing workflow already had provisions ready for the concept of a build. The idea for the download would be a button which "requests" the build and shows the status on the button, and then the button becomes enabled once the build is available.

**Comment by freshollie** [comment]
I basically instructed @riwanou to do this next :)

**Comment by HThuren** [comment]
@freshollie @raphaelcoeffic @riwanou this will be a nice improvement, Cloudbuild where you can have the build as a file, even if you don't have the proper device (then doit yourself). Maybe 2 buttons, Build and Build and flash :-)

**Comment by raphaelcoeffic** [comment]
@HThuren yeah, that's the idea, whoever the boys had something a bit nicer in mind, so it needs a bit more work ;-)
